src/main/kotlin/org/opensearch/replication/action/stop/TransportStopIndexReplicationAction.kt

```diff
@@ -174,6 +194,36 @@ class TransportStopIndexReplicationAction @Inject constructor(transportService:
         return AcknowledgedResponse(inp)
     }
 
+    class RemoveStaleReplicationTasks(val request: StopIndexReplicationRequest, listener: ActionListener<AcknowledgedResponse>) :
```

Comment:
Can we re-use transport action under persistent task service to clear these entries?
and please check for "task not found" case as well.

Reply:
Done